Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.ts

```diff
@@ -136,6 +136,31 @@ namespace gdjs {
       this.invalidateBounds();
     }
 
+    drawChamferRectangle(
+      x1: float, 
+      y1: float, 
+      x2: float, 
+      y2: float, 
+      chamfer: float
+    ){
+      this.updateOutline();
+      this._graphics.beginFill(
+        this._object._fillColor,
+        this._object._fillOpacity / 255
+      );
+      //@ts-ignore from @pixi/graphics-extras
+      this._graphics.drawChamferRect(
+        x1, 
+        y1, 
+        x2-x1,
+        y2-y1,
+        chamfer ? chamfer : 1
```

Comment:
I checked the code in PIXI (https://api.pixijs.io/@pixi/graphics-extras/src/drawChamferRect.ts.html) and you can see that if `chamfer <= 0`, a rectangle is drawn. So we could allow chamfer to take any value here.